src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -1169,6 +1169,47 @@ void uct_rc_mlx5_iface_common_event_cq(uct_ib_iface_t *ib_iface,
     iface->cq[dir].cq_sn++;
 }
 
+static UCS_F_ALWAYS_INLINE void
+uct_rc_mlx5_iface_emplace_compressed_cqes(uct_ib_mlx5_cq_t *cq,
+                                          unsigned cqe_title_index)
+{
+    uct_ib_mlx5_cq_decompress_t *cq_decompress
+                                = &cq->cq_decompress;
+    struct mlx5_cqe64 *title    = uct_ib_mlx5_get_cqe(cq, cqe_title_index);
+    int cqe_pos, next_arr_index = cqe_title_index + 1;
+
+    struct mlx5_cqe64 *mini_cqe_arr;
+    struct mlx5_cqe64 *cqe;
+    uct_ib_mlx5_mini_cqe8_t *mini_cqe;
+
+    /* Initialize the decompression related data */
+    cq_decompress->block_size   = ntohl(title->byte_cnt);
+    cq_decompress->left         = cq_decompress->block_size;
+    cq_decompress->wqe_counter  = ntohs(title->wqe_counter);
+    cq_decompress->mini_arr_idx = 0;
+    memcpy(&cq_decompress->title, title, sizeof(struct mlx5_cqe64));
```

Comment:
why need special code for rc?
i would expect to reuse the IB decompression code (such as uct_ib_mlx5_iface_cqe_decompress_start/continue)